### DISCUSSION TITLE
Support for Multiple Tags in REST API

### DISCUSSION THREAD

**Discussion by BlafKing** [civitai/civitai#746] [Ideas]
Currently, it seems that the API does not support the ability to filter by multiple tags in a single request. I've tried various common conventions to send an array of tags (comma-separated, repeating the parameter, square brackets, etc.) but none seem to be supported.

Being able to filter by multiple tags in a single request would be really helpful and I'd love to hear if this is either already possible or if it could be implemented.

Thanks! :)

**Comment by Pemmican6** [comment]
I would also like to see this implemented, as it is a fairly standard feature, and the front end supports multiple tag searching.

**Comment by getsmartt** [comment]
API does allow multiple tags (for images, haven't looked at anything else).  Comma separated works, but it it uses a IN statement, so it is images that have TagA OR TagB, not images that have BOTH TagA AND TagB.  Having the ability to do TagA AND tagB would be beneficial.
